### PR TITLE
Version 0.1

### PR DESCRIPTION
**Pull request by cberube**
This package provides the base interface for a _status stream_, which is similar in some sense to a logger implementation but is designed for process state information that is targeted to a real-time (or near-real-time display), like a dashboard.

Convenience methods for four standard states are provided, along with a general `update` method which allows custom states to be set.

Any state update message may also include a context, which may be used by the code receiving the updates to display additional details about the process.

### CODE REVIEW

**Comment by nwarlen** [comment]
travis.yml?

**Comment by cberube** [comment]
This really shouldn't have a travis.yml file yet -- I flipped it on in Travis too soon (there's actually no executable code, so... there's nothing to build or test at the moment)

**Comment by nwarlen** [comment]
Makes sense, the red `x` just gave me chills :stuck_out_tongue_closed_eyes:

**Comment by cberube** [comment]
@nwarlen as well it should :+1:

File: src/StatusStreamInterface.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{
+    /**
+     * Updates the current state to `active`, possibly with some context data
+     *
+     * @param mixed|null $context
+     */
+    public function active($context = null);
+
+    /**
+     * Updates the current state to `degraded`, possibly with some context data
```
Comment (nackjicholson) [inline src/StatusStreamInterface.php:15]: What does degraded state mean?

Comment (cberube) [inline src/StatusStreamInterface.php:15]: Operating in a way that is not ideal, but recoverable (e.g. if the digest system tried to process data for a measurable but the measurable was locked by another digest process that might be a temporary "degraded" state -- it's normal and recoverable, but means that the process is delayed).

**Comment by cberube** [comment]
Closing this PR to move to version 0.2

**Pull request closed without merging**